css/style.css

```diff
@@ -50,8 +56,8 @@ a {
   display: none;
 }
 
-.menu-btn {
-  margin-top: 5px;
+.menu-btn .fa-bars {
+  font-size: 25px;
 }
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure on the desktop we are not seeing the navbar when the page refreshes. The menu is supposed to be visible only on the mobile version. Kindly correct that... :warning: :warning:  :warning: 

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201905451-af2fb8f4-2d50-41a8-8fb1-8aef47caf690.png)